app/Http/Controllers/QuizAPIController.php

```diff
@@ -23,4 +24,27 @@ public function displayAllQuizzes():JsonResponse
             "data" => $quizzes
         ]);
     }
+
+    public function addNewQuiz(Request $request): JsonResponse
+    {
+        $request->validate([
+            'name' => 'required|string|min:1|max:10',
+            'description' => 'required|string|min:1|max:100'
```

Comment:
Why are these values so small when your database can hold 255 characters for a name and 1000 for a description?